tests/e2e/configurer/upgrade.go

```diff
@@ -127,14 +128,20 @@ func (uc *UpgradeConfigurer) CreatePreUpgradeState() error {
 
 	chainANode.CreateBalancerPool("pool1A.json", initialization.ValidatorWalletName)
 	chainBNode.CreateBalancerPool("pool1B.json", initialization.ValidatorWalletName)
+	stableswapPoolIdA := chainANode.CreateStableswapPool("stablePool.json", initialization.ValidatorWalletName)
+	chainBNode.CreateStableswapPool("stablePool.json", initialization.ValidatorWalletName)
 
 	// enable superfluid assets on chainA
 	chainA.EnableSuperfluidAsset("gamm/pool/1")
 
 	// setup wallets and send gamm tokens to these wallets (only chainA)
-	lockupWalletAddrA, lockupWalletSuperfluidAddrA := chainANode.CreateWallet(lockupWallet), chainANode.CreateWallet(lockupWalletSuperfluid)
+	lockupWalletAddrA, lockupWalletSuperfluidAddrA, stableswapWalletAddrA := chainANode.CreateWallet(lockupWallet), chainANode.CreateWallet(lockupWalletSuperfluid), chainANode.CreateWallet(stableswapWallet)
 	chainANode.BankSend("10000000000000000000gamm/pool/1", chainA.NodeConfigs[0].PublicAddress, lockupWalletAddrA)
 	chainANode.BankSend("10000000000000000000gamm/pool/1", chainA.NodeConfigs[0].PublicAddress, lockupWalletSuperfluidAddrA)
+	chainANode.BankSend("100000stake", chainA.NodeConfigs[0].PublicAddress, stableswapWalletAddrA)
+
+	// test swap exact amount in for stable swap pool (only chainA)A
+	chainANode.SwapExactAmountIn("2000stake", "1", fmt.Sprintf("%d", stableswapPoolIdA), "uosmo", stableswapWalletAddrA)
```

Comment:
We should also have a separate test fixture for swapping against a stableswap pool in `e2e_test.go` (post-upgrade)

Reply:
Got it

Reply:
Hii. Any updates here?

Reply:
@p0mvn I added new test on e2e_test file. please check